### PR TITLE
Updated readme.md to make installation easier

### PR DESCRIPTION
**Pull request by skynet01**
- it took me an extra hour to install this script for the first time (never used Appdaemon before) even though I consider myself an advanced user. 
- I think more clear directions would make it easier. Created requirements and step by step installation section using HACS as I take it this will be the case for most people :)
- Reused as much of your content as I could, created a simple list to follow :)

### CODE REVIEW

**Comment by Odianosen25** [comment]
Hello @skynet01,

Thanks for this. For some reasons I didn't see this come in, and wouldn't have if not someone just submitted an issue. Been very busy of late anyway. 

Can you kindly redo it against the dev?

Kind regards

**Review by skynet01 — COMMENTED** [review]
> Hello @skynet01,
> 
> Thanks for this. For some reasons I didn't see this come in, and wouldn't have if not someone just submitted an issue. Been very busy of late anyway.
> 
> Can you kindly redo it against the dev?
> 
> Kind regards

done

File: README.md [hunk 1]
```diff
@@ -36,34 +36,53 @@ This App is designed to maximise the use of the detection system, so that the us
...
+-------------------------------------------------------------------------- 
+- [Home Assistant](https://www.home-assistant.io/getting-started/) 
+- [MQTT Broker](https://www.home-assistant.io/docs/mqtt/broker/) Mosquitto MQTT broker add-on from Add-on-Store works out of the box
+- [Appdaemon](https://appdaemon.readthedocs.io/en/latest/INSTALL.html) >= 4.0 running (of course :roll_eyes:). You can install AppDaemon addon from the Add-on-store. Make sure to also [enable MQTT plugin in Appdaemon](https://appdaemon.readthedocs.io/en/latest/CONFIGURE.html#configuration-of-the-mqtt-plugin). 
+    - A simple AppDaemon plugin configuration sufficient for this app, in the `appdaemon.yaml` file is seen below.
     ```yaml
     plugins:
         HASS:
           type: hass
-          ha_url: <some_url>
```
[the remark this answers is not in the document]
Comment (Odianosen25) [reply README.md:54]: Well if working outside hassio, there is need for it. But I understand a lot of users might be hassio based, so will leave it as is.

Thanks for the editing.

Regards

[the remark this answers is not in the document]
Comment (Odianosen25) [reply README.md:54]: @skynet01, 

I just checked and it seems the PR is still against the master branch and not dev.

I think you might need to close this one, and create a new one, as I know not if its possible to move the PR against another branch?

Regards

**Comment by skynet01** [comment]
I just closed this pull request, you can see it now in dev branch

**Pull request closed without merging**